### PR TITLE
update repository name in files

### PR DESCRIPTION
**Pull request by feedmypixel**
- Update the repo name change
- Move author into contributors
- Add new staff to contributors
- Add TOC to README.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,7 +95,12 @@
     "lint:sass:modified",
     "lint:js:modified"
   ],
-  "author": "Zac Tolley <zac@thetolleys.com>",
+  "contributors": [
+    "Ben Chidgey <ben@feedmypixel.com> (http://feedmypixel.com)",
+    "Dom Smith <dom@teneightfive.com> (http://github.com/teneightfive)",
+    "Tyom Semonov",
```
Comment (tyom) [inline package.json]: `https://github.com/tyom`

File: README.md [hunk 2]
```diff
@@ -10,6 +10,26 @@ access it, no matter what their limitation.
 In order to use the application the front end layer must be run, with a small number of settings,
 and be provided with a back end server to provide the API, data storage and search engine capabilities.
 
+## Table of Contents
```
Comment (teneightfive) [inline README.md]: 👍 Just need to remember to update this when we update the readme in future.

Comment (feedmypixel) [inline README.md]: 👍  https://github.com/thlorenz/doctoc

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Data hub Beta 2
+# Data hub frontend
```
Comment (tyom) [inline README.md]: Can we update it to "Data Hub" whilst we're at it?

**Pull request merged**